larpandoradlcontent/LArHelpers/LArDLHelper.cc

```diff
@@ -0,0 +1,49 @@
+/**
+ *  @file   larpandoradlcontent/LArHelpers/LArDLHelper.cc
+ *
+ *  @brief  Implementation of the lar deep learning helper helper class.
+ *
+ *  $Log: $
+ */
+
+#include <larpandoradlcontent/LArHelpers/LArDLHelper.h>
+
+namespace lar_content
+{
+
+using namespace pandora;
+
+StatusCode LArDLHelper::LoadModel(const std::string &filename, LArDLHelper::TorchModel &model)
+{
+    try
+    {
+        model = torch::jit::load(filename);
+        std::cout << "Loaded the TorchScript model \'" << filename << "\'" << std::endl;
+    }
+    catch (...)
```

Comment:
Should we catch a specific torch exception here, or is catching everything just the safest and most sensible thing?

Reply:
I think this is safer. It's actually quite difficult to work out what the full set of exceptions here is, so given that this is file loading and any exception is likely to be fatal, I think this is the safest approach.